var/spack/repos/builtin/packages/mkl/package.py

```diff
@@ -58,6 +58,7 @@ def setup_dependent_package(self, module, dspec):
         # different interface and threading layers.
 
         name = 'libmkl_rt.%s' % dso_suffix
+        # FIXME : change the logic here
```

Comment:
i think the basic list of names is: `mkl_intel_lp64;mkl_sequential;mkl_core' plus system's `pthread;m`,

where the first two change based on 32/64 ints and threaded or not.